### DISCUSSION TITLE
Volume Display

### DISCUSSION THREAD

**Discussion by pablo-hp** [File-New-Project/EarTrumpet#1457] [Ideas]
Currently, I have configured the following commands to increase and decrease the volume.
"Ctrl+Shift+Up" and "Ctrl+Shift+Down".
Everything works very well.

But I would like that when using the commands on the keyboard, information about the percentage of the currently defined volume would be shown on the screen at the same time.
This currently occurs when holding the mouse over the volume icon.
It shows the volume percentage.

But when using the command on the keyboard, we do not have any visual information about the adjusted volume percentage.

So, I would like to suggest some type of visual information on the screen, about the adjusted volume value, when using the keyboard.
I don't know if I explained it correctly, I hope so.

Grateful.

**Comment by zedd-0** [comment]
an OSD for system volume changing would be nice.

maybe a little advanced, 
with information about brightness 
and device connections (USB stick), too.
:P

**Comment by at-ng** [comment]
I think this https://github.com/File-New-Project/EarTrumpet/discussions/672 would be a nice way to do it.
